.stylelintignore

```diff
@@ -0,0 +1,4 @@
+project-base/src/**/helpers/*.*
+project-base/src/**/libs/*.*
+project-base/src/**/libs/**/*.*
+project-base/src/**/core/reset.less
```

Comment:
Do we need this file at all? If so, please add newline at the end of file

Reply:
Sure, I forgot commit it. Sorry.

Reply:
Sure, I forgot commit it. We don't need this.